common/djangoapps/student/tests/test_reset_password.py

```diff
@@ -113,6 +119,10 @@ def test_reset_password_email(self, send_email):
         self.assertEquals(len(to_addrs), 1)
         self.assertIn(self.user.email, to_addrs)
 
+        self.assert_event_emitted(
+            SETTING_CHANGE_INITIATED, old=None, new=None, user_id=self.user.id, setting=u'password'
```

Comment:
I'm wondering if it would be clearer if you dropped the old and new parameters since they will default to None (I think). If you disagree, I'd prefer that the values appear at the end after the setting parameter so that it is clearer what they relate to.